docs/sections/managing_data/groups.md

```diff
@@ -132,15 +132,15 @@ Group description  <no description>
 # Nodes:
 PK    Type         Created
 ----  -----------  -----------------
-74  CalcJobNode  2078D:17h:51m ago
-76  CalcJobNode  2078D:17h:57m ago
+ 380  CalcJobNode  2078D:17h:46m ago
+1273  CalcJobNode  2078D:18h:03m ago
 ...
 ```
 
 Removing nodes from the group is similar to adding them, try:
 
 ```{code-block} console
-$ verdi group remove-nodes -G my_group <PK>
+$ verdi group remove-nodes -G my_group 380
```

Comment:
Throughout the tutorial, we try to just put `<PK>`, so it's clear they have to change this `<PK>` by the PK of their database. I'd stick to using that, since it's likely the PK here won't match theirs.

Reply:
that's fair. So would you agree to unify those in line 110 above to use `<PK1>`, `<PK2>`?

Reply:
I'd just stick with `<PK>`?